### ISSUE TITLE
Relationship diagram does not include column types

### ISSUE THREAD

**Issue by saurabhgour** [schemaspy/schemaspy#625]
The relationship diagram does not include the column type (varchar, int8) etc.. nor could I find a command line argument to make this happen.

**Comment by npetzall** [comment]
No, it doesn't. Most relationship diagrams would become to big. But I could have look at how much work it would be to add it as an option.

**Comment by plc-dev** [comment]
Have there been any advances in this direction? I would very much be interested in this feature as well.

**Comment by Bessonov** [comment]
I'm not sure that the type consume much space. It could use a smaller font size. But an option would be fine. Beside of the type it would be nice to see whether the column is nullable.

**Comment by salvisb** [comment]
+1, I would also appreciate such feature.
